tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG=$(grep "git clone --single-branch" Documentation/Getting-Started/quickstart.md | awk '{ print $6 }')
```

Comment:
Instead of using the quickstart guide, I'm thinking it will be more reliable in the long term to parse the images.txt, something like this:
```suggestion
FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
```